src/components/CountriesList.module.scss

```diff
@@ -3,3 +3,7 @@
   color: goldenrod;
   text-align: center;
 }
+
+.link {
+  all: unset;
```

Comment:
What is the purpose of this style?

Reply:
Lint unset to remove unnecessary blue underlining
3  and  4 look better with style I think, but if it's not I can remove theme